worker-core/src/test/java/com/hpe/caf/worker/core/WorkerCoreTest.java

```diff
@@ -190,7 +190,7 @@ public void testAbortTasks()
     {
         BlockingQueue<byte[]> q = new LinkedBlockingQueue<>();
         Codec codec = new JsonCodec();
-        WorkerThreadPool wtp = WorkerThreadPool.create(2);
+        WorkerThreadPool wtp = WorkerThreadPool.create(2, () -> System.out.println("Hit runnable due to interruption"));
```

Comment:
Removed override of default handler as throwable handler would not get invoked in this test